controllers/application/ingress_gateway.go

```diff
@@ -134,3 +134,8 @@ func isIngressGateway(gateway *networkingv1beta1.Gateway) bool {
 
 	return match
 }
+
+func hasIgnoreLabel(obj client.Object) bool {
+	labels := obj.GetLabels()
+	return labels["skiperator.kartverket.no/ignore"] == "true"
+}
```

Comment:
This should probably be placed somewhere else instead of under `ingress_gateway.go`

Reply:
TODO: Check naming convention